pkg/daemon/ceph/osd/volume.go

```diff
@@ -696,3 +696,30 @@ func getCephVolumeRawOSDs(context *clusterd.Context, clusterName string, cephfsi
 
 	return osds, nil
 }
+
+func callCephVolume(context *clusterd.Context, args ...string) (string, error) {
+	// Use stdbuf to capture the python output buffer such that we can write to the pod log as the
+	// logging happens instead of using the default buffering that will log everything after
+	// ceph-volume exits
+	baseCommand := "stdbuf"
+	// TODO: what if log path were /dev/stderr and we didn't do combined output so that logs would
+	//       appear in the prepare pod log?
+	baseArgs := []string{"-oL", cephVolumeCmd} // , "--log-path", cvLogDir}
+
+	co, err := context.Executor.ExecuteCommandWithCombinedOutput(baseCommand, append(baseArgs, args...)...)
+	if err != nil {
+		// Print c-v log before exiting with failure
+		// TODO: this is just the default log location; should this be a tmp location so the log
+		//       doesn't grow large on disk after many runs?
```

Comment:
Open question. Thoughts @travisn / @leseb ?

Reply:
Is /var/log/ceph mapped to the host? Agreed we wouldn't want the log to keep growing every time the prepare job is run. @leseb Is the c-v log rotated or otherwise cleared out ever?

Reply:
Typically the logs are exposed on the host and not rotated. I wouldn't be concerned with the size of that log. It's `208K` on my system for 3 OSDs.

Reply:
What are your thoughts compared to instead directing this log to stderr so it gets to be part of the pod log? Or would that be bad for the Ceph log collector?

Reply:
As I recall @leseb  and I discussed this in the past and it was just a question of whether it would be too verbose for the pod log. But if we just dumped the log at the end before exiting the job, it would certainly be easier to troubleshoot if the log was always available.